STEERING-COMMITTEE.md

```diff
@@ -48,6 +48,37 @@ the way we run this committee, based on feedback from the community.
 1. Act as the final escalation point and decider for any disputes, issues,
    clarifications, or escalations within the project scope.
 
+## Knative Trademark Committee Charter
+
+1. Manage licensing the Knative brand and decide which things can be called
+   "Knative" and how that mark can be used in relation to other efforts or
+   vendors.
+1. Review and approve conformance rules and tests that define brand use
+   decisions.
+1. Review and approve definition of Knative Core scope.
+1. Only the Trademark Committee can redefine its own charter and authority over
+   Knative Trademark, conformance, and brand decisions.
+
+## Trademark Committee Composition
+
+The Trademark Committee is composed of company representatives from companies
+which have substantially contributed to Knative over the last year. For the 2020
+year, one seat is assigned to each of Google, IBM/RedHat, and VMware. The
+committee shall review this composition at least annually and update these
+rules.
+
+The Trademark Committee desires to always reach consensus. Additionally, each
+member can veto decisions regarding conformance and scope.
+
+Decisions are made in meetings when a quorum of the members are present and may
+pass with at least half the members of the committee supporting it.
+
+Quorum is considered reached when at least of the members are present.
+
+In case of extended absence (or removal from the project by the steering
+committee), the organization of the absent member may appoint a single
+replacement delegate from the same company during the absence.
+
```

Comment:
Something that isn't clear here, which we should be explicit about is this: this committee should meet in the open, recorded, and the community should be welcome to attend and provide input.

While "binding" votes seem to be allocated based on the previous year's contributions, there are smaller contributors that should have a voice in these discussions (think "non-binding" votes).

I think @vaikas had a similar comment on the doc, and I wanted to make sure this wasn't lost.

Reply:
I think legal action or recommendations aren't really suitable for public forums. What is the value of a public forum for these discussions?

Perhaps there's a middle ground where discussions that are of broad interest -- like conformance or  scope decisions -- are different from compliance/legal discussions. It is too easy for careless words to affect the integrity of a trademark.

Reply:
You could have an established cadence of public and private meetings, like the CNCF TOC does.

Reply:
> I think legal action or recommendations aren't really suitable for public forums

Ack, but I would expect all discussion of scope, conformance, voting and vetoing to happen in the open and encourage community input on the discussion.  I generally suspect that this will be the vast majority of discussions, and I'm sort of skeptical that there will be sufficient legal action to warrant regular private meetings until the project grows considerably.  Doing private meetings one-off for these exceptional cases seems perfectly reasonable.

Reply:
I copied the proposed Steering text here, but didn't specific a meeting frequency.

Reply:
(The steering text is that at least the decisions should be recorded publicly, and the meetings should default to open.)